lib/src/main/kotlin/at/bitfire/synctools/mapping/calendar/builder/TitleBuilder.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * This file is part of bitfireAT/synctools which is released under GPLv3.
+ * Copyright © All Contributors. See the LICENSE and AUTHOR files in the root directory for details.
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+package at.bitfire.synctools.mapping.calendar.builder
+
+import android.content.Entity
+import android.provider.CalendarContract.Events
+import at.bitfire.ical4android.Event
+import at.bitfire.vcard4android.Utils.trimToNull
+
+class TitleBuilder: AndroidEventFieldBuilder {
+
+    override fun build(from: Event, main: Event, to: Entity) {
+        to.entityValues.put(Events.TITLE, from.summary.trimToNull())
```

Comment:
We could check for null and only put if it's not null, would that not be cleaner since the entity value of title always starts out being null? Or do we have to overwrite an existing value with null sometimes?